packages/assets/profiles/samsung/samsung-gearvr.json

```diff
@@ -0,0 +1,66 @@
+{
+    "profileId" : "samsung-gearvr",
+    "assets" : {
+        "none" : {
+            "path" : "none.glb",
+            "rootNodeName" : "none-controller-node"
+        }
+    },
+    "layouts": {
+        "none": {
```

Comment:
This should define a "left-right-none" the same way that the oculus-go and google-daydream profiles do, because while it's not an inherently handed device the platform will assign it a hand and we'd expect that to be communicated by the API.